assets/designer/environments/designer-pytorch-2.3-train/context/conda_dependencies.yaml

```diff
@@ -5,17 +5,16 @@ channels:
 dependencies:
   - cudatoolkit=11.8
   - pip=20.2
-  - python=3.8.10
+  - python=3.9.18
   - pytorch=2.3.1
   - torchvision=0.18.1
   - pip:
-      - azureml-designer-pytorch-modules==0.0.69
+      - azureml-designer-pytorch-modules==0.0.71
       - future==0.18.3
       - numpy==1.22
-      - cryptography==42.0.4
       - azureml-dataprep==4.10.1
-      - Werkzeug==3.0.3
-      - pip==21.1
+      - Werkzeug==3.0.6
+      - pip==23.3
```

Comment:
pip should be defined as conda dependency, just update version above line#7

Reply:
just like the comments in #3625, We need pip 20.2, which uses the legacy resolver, to install our package.